lib/appsignal/probes/puma.rb

```diff
@@ -0,0 +1,60 @@
+module Appsignal
+  module Probes
+    class PumaProbe
```

Comment:
Shall we drop the `Probe` suffix to these classes or not?

Reply:
I'd say keep it, removes any ambiguity from the naming.